app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<section class="form-container" style="padding: 0 15px;">
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <nav>
+    <%= link_to raw('&larr;'), :back, class: "arrow-link" %>
+    <h2>REGISTER</h2>
+  </nav>
+
+  <div class="field">
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: "Full name" %>
+  </div>
+
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email" %>
+  </div>
+
+  <div class="field">
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", placeholder: "Password" %>
+  </div>
+
+  <div class="field">
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: "Password Confirmation" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign Up", class: "btn btn-primary" %>
+  </div>
+<% end %>
```

Comment:
- [ ] [OPTIONAL]
Nice job implementing requested changes, however, I think other aspects of your signup page design got affected negatively. Can we improve upon these aspects of the design?
  * The blue nav area should stretch across the entire width of the screen, with proportionate height, like seen in the template design.
  * The form fields should be similar the template design version, you can maintain the signup button you added below, but it should look bold and well-spaced accordingly.


**Your Register page:**       |  **Expected Register page:**     
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/Megagig/Budget-App/assets/12745474/52bcb4fa-426b-4700-8a8a-7be5657ee78e"> | <img width="800" alt="Home Page" src="https://github.com/Megagig/Budget-App/assets/12745474/3338e141-a039-4e4f-9e21-7a80bc6de2cc">